### ISSUE TITLE
Global Scope Cleanup

### ISSUE THREAD

**Issue by SyntaxViking** [DinoDevs/GladiatusCrazyAddon#490]
Gents,

Currently, many functionalities in the addon are placed in the global scope without clear categorization, making navigation somewhat difficult & illogical at times. To improve usability and maintainability, I propose reorganizing some of the features into relevant categories. Below are the suggested changes:

#Global Scope Cleanup (from top to bottom)

Display auction status bar -> Auction page
Alert when auction status changes -> Auction page
Remember merchants tabs -> Merchants page
Show attacked timers -> Arena page

Notify me when there is a new guild application -> Guild page
Show guild pinned messages from the baths in messages -> Guild page
Check for guild applications and pinned messages every (minutes) -> Guild page
Notify me when guild war attack cooldown -> Guild page
Check the guild war cooldown every (minutes) -> Guild page

Enable items shadows -> Accessibility page
Group inventory options -> Accessibility page
Show inventory items' gold price -> Accessibility page
Change pages-box's layout -> Accessibility page
Show gold and exp data -> Accessibility page

Display pray shortcut when in Underworld -> Expedition page

Display durability on item's bottom-left corner -> Accessibility page
Notification for items with durability+conditioning under _% -> Accessibility page
Display item's forge materials on tooltips -> Accessibility page
Display mercenaries real names (type) and combat stats on tooltips -> Accessibility page
Display buff values on reinforcements & upgrades -> Accessibility page

Display Global Arena timer -> Arena page

**Comment by FrutyX** [comment]
I'll be honest with you, GCA is more like on a life-support mode, time to time I add something, but only when I have the time and mood for it.

Those features you listed, they are not global without a reason, they run (globally) on every page, not specific pages (some features must always run, otherwise they would not show, logically). 

And unfortunately, you can't simply move something into a different category, you have to change ten other things, so you don't break anything that's is supposed to be there.

I don't see "navigation difficult" tho, Ctrl+F is a functionality you'll find everywhere nowadays, talking code-wise. Talking about user settings, the settings aren't that huge.

I can slap an "Enhancement" label on this, but there are no promises.

**Comment by GreatApo** [comment]
@SyntaxViking you are right. As @FrutyX mentioned, that those run globally and they were added in the global section mainly because GCA slowed down the browser a lot if all those options were enabled. As time passed and most people upgraded to newer computers, this became less of a problem.

We should probably move these options, but maybe add a footnote that they run across all game pages.

**Comment by FrutyX** [comment]
Hello!

This Issue has been selected to be merged into our Feature Requests & Enhancements overview Issue -  #530.

Please note that this does not change anything, the request is still taken in account, but it's moved for better and cleaner GitHub management.

Thank you for your feedback and ideas! We appreciate them :).
